cypress/utils/utils.ts

```diff
@@ -368,32 +368,35 @@ export function applySearchFilter(
     cy.wait(4000);
 }
 
-export function sortAsc(
-    sortCriteria: string,
-    tableSelector = `th[data-label="${sortCriteria}"]`
-): void {
-    cy.get(tableSelector).then(($tableHeader) => {
-        if (
-            $tableHeader.attr("aria-sort") === "descending" ||
-            $tableHeader.attr("aria-sort") === "none"
-        ) {
-            $tableHeader.find("button").trigger("click");
-        }
-    });
+//tableSelector should be .pf-c-table__sort as the fieldHeader in views
+export function sortAsc(sortCriteria: string, tableSelector: string): void {
+    cy.get(tableSelector)
+        .contains("th", sortCriteria)
+        .then(($tableHeader) => {
+            const button = $tableHeader.find("button");
+            if (
+                $tableHeader.attr("aria-sort") === "none" ||
+                $tableHeader.attr("aria-sort") === "descending"
+            ) {
+                button.trigger("click");
+            }
+            cy.wrap($tableHeader).should("have.attr", "aria-sort", "ascending");
+        });
 }
 
-export function sortDesc(
-    sortCriteria: string,
-    tableSelector = `th[data-label="${sortCriteria}"]`
-): void {
-    cy.get(tableSelector).then(($tableHeader) => {
-        if (
-            $tableHeader.attr("aria-sort") === "ascending" ||
-            $tableHeader.attr("aria-sort") === "none"
-        ) {
-            $tableHeader.find("button").trigger("click");
-        }
-    });
+export function sortDesc(sortCriteria: string, tableSelector: string): void {
+    cy.get(tableSelector)
```

Comment:
with simple if and else these two methods can be combined into one method .

````
export function sortAscOrDesc(){

oppositeofsortBy = "ascending" if (sortCriteria) = "descending" else "ascending" 



if (
                $tableHeader.attr("aria-sort") === "none" ||
                $tableHeader.attr("aria-sort") === sortCriteria

cy.wrap($tableHeader).should("have.attr", "aria-sort", oppositeofsortBy);
```
`

Reply:
I have implemented this in PR https://github.com/konveyor/tackle-ui-tests/pull/562 . Please use 
clickOnSortButton() method instead .